app/Http/Controllers/AuthController.php

```diff
@@ -176,4 +177,23 @@ public function reset_password(Request $request)
             'message' => trans('responses.reset_password'),
         ]);
     }
+
+    /*
+     * Logout the current user session 
+     *
+     * @return Illuminate\Http\JsonResponse
+     */
+    public function logout()
+    {
+        if (!Auth::check())
+            return response([
+                'message' => trans('responses.AuthController.logout.error')
```

Comment:
Is this necessary? I mean, the api middleware check if the token is active

Reply:
Yeah, I ran tests again and everything is working, you were right